.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!-- Thank you for contributing a pull request! Here are a few tips to help you:
+
+1. If this is your first contribution, make sure you've read the Contribution Checklist https://developers.mattermost.com/contribute/getting-started/contribution-checklist/
+2. Read our blog post about "Submitting Great PRs" https://developers.mattermost.com/blog/2019-01-24-submitting-great-prs
+3. Take a look at other repository specific documentation at https://developers.mattermost.com/contribute
+-->
+
+#### Summary
+<!--
+A description of what this pull request does.
+-->
+
+#### Ticket Link
+<!--
+If this pull request addresses a Help Wanted ticket, please link the relevant GitHub issue, e.g.
+
+  Fixes https://github.com/mattermost/mattermost-server/issues/XXXXX
+
+Otherwise, link the Jira ticket.
+-->
+
+#### Checklist
+<!--
+Please review each item, and remove those not relevant for your pull request
+-->
+- Tested related changes <!-- See https://developers.mattermost.com/contribute/server/developer-workflow/#running-only-specific-server-unit-tests how to run specific unit tests. -->
```

Comment:
Intentionally omitting the `[ ]` for checkboxes?

Reply:
Yes. The old checklist had some unnecessary steps IMO. You would check every item, remove the unrelated ones and then also check the boxes. IMO the last step is unneeded.

Reply:
Added them again does feedback.